### DISCUSSION TITLE
New TEST release tau-argus

### DISCUSSION THREAD

**Discussion by ppdewolf** [sdcTools/UserSupport#240] [Team Testers]
A new release for testing is uploaded: [Tau-Argus version 4.2.4](https://github.com/sdcTools/tauargus/releases/tag/v4.2.4-TEST).  

Please let me know whether you are able to test this version. The main new features to be tested are:
- Save and read recode files, including recode files for tree-based hierarchical variables
- Keep status (optionally) with tabular input made additive

Additional information on the new features can be found in the `Version history` section of the `ReleasenoteTauArgus4.2.4.pdf`.

Obviously, other tests are welcome as well, e.g. making sure that older features are still working as before.

Please let me know whether you are able to test this version. I would like some feedback on tests before March 25, because I would like to make the release official by April 1.

**Comment by KarinaKelleher** [comment]
Hi Peter-Paul,

I wasn't really in a position to test the new features (under time constraints and not currently using these functions) but did test this new version on our normal processes and everything did work as expected,

Karina.
